biome.json

```diff
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+	"organizeImports": {
+		"enabled": true
+	},
+	"formatter": {
+		"include": ["**/*.ts", "**/*.tsx"],
+		"enabled": true,
+		"formatWithErrors": true,
+		"indentStyle": "tab",
```

Comment:
This won't work, can we use spaces, with width 2?


If using tab, width should be 1, and it will consider the visualization configuration for tabs on the editor. Spaces seemed more reliable to me, while looking better when reviewing here on github

WDYT?

Reply:
Sounds good! Lemme change to spaces